### PR TITLE
Fetch results from the API

### LINKED ISSUE #391: Load discovery add-ons from the API
We need to load the add-ons from the API instead of using our fake data. This depends on the API in https://github.com/mozilla/addons-server/issues/2447 and should use the locale from #147.

### PR DESCRIPTION
**Pull request by mstriemer**
There appears to be a bug where the icon doesn't go away on uninstall. This was working previously so I'm not sure what changed. It's definitely uninstalled...

disco-api mov

Fixes #391.

### CODE REVIEW

File: src/disco/components/Addon.js [hunk 1]
```diff
@@ -77,13 +84,24 @@ export class Addon extends React.Component {
...
+    if (type === THEME_TYPE) {
+      return (
+        <p className="editorial-description">{i18n.gettext('Hover over the image to preview')}</p>
+      );
+    }
+    return (
+      <div
+        ref="editorialDescription"
+        className="editorial-description"
+        dangerouslySetInnerHTML={sanitizeHTML(description, ['blockquote', 'cite'])} />
```
Comment (muffinresearch) [inline src/disco/components/Addon.js]: Would be nice to have some tests just to prove this is working as we expect.

File: karma.conf.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ const newWebpackConfig = Object.assign({}, webpackConfigProd, {
     new webpack.NormalModuleReplacementPlugin(/config$/, 'core/client/config.js'),
     // Substitutes client only config.
     new webpack.NormalModuleReplacementPlugin(/core\/logger$/, 'core/client/logger.js'),
+    // Use the browser's window for window.
+    new webpack.NormalModuleReplacementPlugin(/core\/window/, 'core/browserWindow.js'),
```
Comment (muffinresearch) [inline karma.conf.js]: This might make it possible to cover more code via karma we should look at that separately.

**Comment by muffinresearch** [comment]
r+wc with some tests added to directly cover the purification step.

**Comment by andymckay** [comment]
Excited by this, thanks @mstriemer 👍

File: src/disco/components/Addon.js (hunk 1 above)
Comment (mstriemer) [inline src/disco/components/Addon.js]: Ah, yes. We had discussed that but I forgot about it. Thanks!

**Pull request merged**